CHANGELOG.md

```diff
@@ -1,5 +1,112 @@
 # Release Notes
 
+## Version 1.19.0-beta.0 - 2021-04-05
+
+Features:
+
+* add minikube image rm command [#10924](https://github.com/kubernetes/minikube/pull/10924)
```

Comment:
lets add single quote for any command, it show it gray
```
add `minikube image rm` command
```

Reply:
done